src/PageContent/Settings/Settings.js

```diff
@@ -0,0 +1,148 @@
+import AccountManager from './AccountManager';
+import './SCSettings0.css';
+import './SCSettings1.css';
+
+class Settings extends Component {
+
+    constructor(props) {
+        super(props);
+        this.state = {
+            account: null,
+            org: null,
+            coordinator: null,
+            secondaryContact: null,
+            isEditingOrg: false,
+            isEditingAcc: false,
+            email: 'phisigmaro@uw.edu'
+        };
+    }
+
+    // Backend TODO: Get data from DB
+    componentDidMount() {
+        let account  = {
+            user_type: 'admin',
+            name: 'Phi Sigma Ro',
+            email: this.state.email,
+            password: 'password',
+            organizationPhone: '209-992-9292',
+            address: {
+                street1: '4138 Brooklyn Ave NE',
+                street2: '',
+                city: 'Seattle',
+                state: 'WA',
+                zipcode: 98105
+            },
+            isVerified: true,
+            isActivated: true,
+            numVolunteers: 50,
+            coordinator: {
+                name: 'Andy Duncan',
+                email: 'andyd@uw.edu',
+                phone: '206-487-2859',
+                position: 'President'
+            },
+            secondaryContact: {
+                name: 'Jennifer Marly',
+                email: 'jmarly@uw.edu',
+                phone: '206-487-2992',
+                position: 'Vice President'
+            },
+            notifications: [
+                {
+                    type: 'recurring_pickup_request',  // Enums.NotificationType
+                    content: '-L5QoXeC_UrL5tRRED3e'  // key of DeliveryRequest
+                }
+            ],
+            smsNotif: true,
+            emailNotif: true
+        };
+        this.setState({
+            account: account,
+            org: {
+                name: account.name,
+                address: account.address,
+                organizationPhone: account.organizationPhone,
+                numVolunteers: account.numVolunteers || null,
+                notes: account.notes || null,
+                emergencypickup: account.emergencypickup || null
+            },
+            coordinator: {
+                details: account.coordinator
+            },
+            secondaryContact: account.secondaryContact || null
+        });
+    }
+
+    render() {
+        return (
+            <div>
+                {this.state.account ?
+
+                    <div> 
+                        <div className="container">
+                            <OrganizationDetails
+                                account={this.state.org}
+                                isAdmin={this.state.account.user_type}
+                                isEditingOrg={this.state.isEditingOrg}
+                                handleEditOrg={this.handleEditOrg.bind(this)}
+                                handleOrgSave={this.handleOrgSave.bind(this)}
```

Comment:
same, I think the component itself can handle whether it's in edit mode, so no need to pass it down as props

Reply:
Fixed.